### ISSUE TITLE
Unable to view the region 

### ISSUE THREAD

**Issue by pradeep-bose** [intel/ipmctl#14]
Unable to see the regions listed , when executing the command on fedora 28 , with kernel 4.17 and all the pre-requisite software

ipmctl show -region

SocketID             ISetID    PersistentMemoryType  Capacity FreeCapacity HealthState
       0 0xf455da901f248a22 AppDirectNotInterleaved 125.0 GiB      0.0 GiB     Healthy
       0 0x4c5fda905b258a22 AppDirectNotInterleaved 125.0 GiB      0.0 GiB     Healthy
       1 0x9a4bda9016238a22 AppDirectNotInterleaved 125.0 GiB      0.0 GiB     Healthy
       1 0xa055da90fd248a22 AppDirectNotInterleaved 125.0 GiB      0.0 GiB     Healthy

**Comment by juston-li** [comment]
Could you clarify further what you mean by not seeing the regions listed?
Thanks!

**Comment by pradeep-bose** [comment]
equivalent to ndctl list -R

**Comment by juston-li** [comment]
So there is no output when you execute 'ipmctl show -region'?

**Comment by pradeep-bose** [comment]
As u can see in the description. The first column of the output is socket Id. There is no region id column in the output. There are no regions listed in the output

**Comment by juston-li** [comment]
There used to be a pool ID which would be handy when creating namespaces but that no longer is the case, the region isn't used as a target anymore.

The pool ID was also something we numbered in ipmctl ourself, it didn't match the region# in ndctl. We dropped it and now use the ISetID/Cookie so that someone could identify the same region in ndctl. (I now see that ndctl displays the value as decimal while we display hex though..)
